src/DynamoRevit/DynamoRevitApp.cs

```diff
@@ -153,6 +153,11 @@ public static Assembly ResolveAssembly(object sender, ResolveEventArgs args)
                 {
                     return Assembly.LoadFrom(assemblyPath);
                 }
+                assemblyPath = Path.Combine(parentDirectory.FullName, @"Revit_2015\" + new AssemblyName(args.Name).Name + ".dll");
```

Comment:
Shouldn't we use assemblyDirectory instead of hardcoding the folder name Revit_2015??